packages/esm-patient-orders-app/src/hooks/useOrderPrice.ts

```diff
@@ -0,0 +1,27 @@
+import { type OrderPriceData } from '../types/order';
+import { type FetchResponse, fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { useMemo } from 'react';
+
+/*
+This implementation depends on these backend modules
+- fhirproxy
+- stockmanagement
+- billing
+*/
```

Comment:
Have you added them explicitly to the `backendDependencies` key in the `orders` app's `route.json` file?